src/platforms/javascript/common/performance/capturing/automatic.mdx

```diff
@@ -1,19 +1,28 @@
 ---
-title: Automatic Capture
-sidebar_order: 20
-description: "Learn how to configure what performance data is automatically captured."
+title: Add Automatic Instrumentation
```

Comment:
Small nit, because of the length of this title when you go to the sidebar it forces this onto two lines (when it adds the scroll bar).

We could drop the `Add` from both this and `Add Manual Instrumentation` to fix. 

![Screen Recording 2020-12-01 at 04 23 53 PM](https://user-images.githubusercontent.com/155016/100798534-c29aec80-33f1-11eb-9325-217e16dd766a.gif)

Reply:
We can, unless you think it's critical that folks understand that there's an option to *add* these types of instrumentation. Otherwise, it can read as reference or background material. But I agree that it's less than ideal to have this wrap to a second line.

Reply:
Personally I think the add isn't necessary. It's the next logical place to go.

Reply:
I'll remove for both automatic and manual.